js/snippetPreview.js

```diff
@@ -919,6 +941,15 @@ SnippetPreview.prototype.updateProgressBars = function() {
 };
 
 /**
+ * Gets the width of the Snippet Preview to set the Snippet Preview Toggler visibility.
+ * @returns {void}
+ */
+SnippetPreview.prototype.handleWindowSize = debounce( function() {
```

Comment:
The name of the function seems to (mostly) be called when resizing the window. Consider altering the name to `handleWindowResizing` or similar to better reflect when this is called.